### PR TITLE
Fusion filt rerun

### PR DESCRIPTION
**Pull request by runjin326**
#### What scientific question is your analysis addressing?
Re-run of fusion_filtering analyses with v8 histologies and new independent RNA file.

#### What was your approach?
Used v8 histologies and new independent RNA file and re-run the module . 

#### What GitHub issue does your pull request address?
https://github.com/PediatricOpenTargets/ticket-tracker/issues/165

#### Which areas should receive a particularly close look?
The `fusion-putative-oncogenic.tsv` did get `broad_histology` update indicating the new histologies file were indeed used. However, the final number of fusions retained and the `percancergroup` and `persample` file were not changed. 

The table comparing the broad histology changes within these two files is attached here:
[compare_broad.txt](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/files/7015765/compare_broad.txt)

#### Is there anything that you want to discuss further?
1) In the file - I noticed that `Embryonal tumors` and `Embryonal tumor`; `Mesenchymal non-meningothelial tumor` and `Mesenchymal, non-meningothelial tumors` were listed as different broad_histology. I believe it's because the most recent TARGET broad_histology update we added "s" but previously we did not use "s" - we might want to fix that later. 

2) I will refer back to v6 (https://github.com/PediatricOpenTargets/ticket-tracker/issues/129) and see whether difference between v6 and v7 are expected to remained filtered in v8. 

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes. 

#### What types of results are included (e.g., table, figure)?
TSV files. 

#### What is your summary of the results?
```
   66601 FilteredFusion.tsv
   30071 fusion-putative-oncogenic.tsv
     133 fusion-recurrent-fusion-bycancergroup.tsv
   11995 fusion-recurrent-fusion-bysample.tsv
     282 fusion-recurrently-fused-genes-bycancergroup.tsv
   11995 fusion-recurrently-fused-genes-bysample.tsv
  121077 total
```

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by runjin326** [comment]
Currently checking whether the changes are expected with new hist + RNA independent list

**Comment by jharenza** [comment]
>Mesenchymal non-meningothelial tumor and Mesenchymal, non-meningothelial tumors were listed as different broad_histology. I believe it's because the most recent TARGET broad_histology update we added "s" but previously we did not use "s" - we might want to fix that later.

Can you open a ticket and let's take care of this now, before the release - these are the things we want to catch before releasing :) - cc @ewafula for an incoming update

**Comment by runjin326** [comment]
@jharenza, I have now looked at the files and the differences are summarized in this PDF file:
[QC_fusion_filtering.pdf](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/files/7022775/QC_fusion_filtering.pdf). Briefly, total `fusion-putative-oncogenic.tsv` does not change and the diff that was loaded on GitHub is due to `broad_histology` changes. 
The diff in `fusion-recurrent-fusion-bysample.tsv` is from new independent RNA sample list used - and when I look at participants, no difference was seen.
The diff in `fusion-recurrent-fusion-bycancergroup.tsv` was slightly puzzling to me - but seems like the differences are all in AML, ALL and NBL, which are what we have changed in this histology file. 
In summary, looks like the merge of some `broad_histology` terms does not affect the fusion list we got and `fusion-putative-oncogenic.tsv` only had `Sample` and `broad_histology` changes.

**Review by jharenza — APPROVED** [review]
thanks for the update! will approve this now, but I think we still ought to go back and check those oncogenic fusions which come up in the filtering of >4 broad histologies (eg _JAK2--JAK2_) - will you write up a ticket for that (lower priority) so we can determine if we need to add a scavenge for those?

**Comment by runjin326** [comment]
> thanks for the update! will approve this now, but I think we still ought to go back and check those oncogenic fusions which come up in the filtering of >4 broad histologies (eg _JAK2--JAK2_) - will you write up a ticket for that (lower priority) so we can determine if we need to add a scavenge for those?
Sure! We had a ticket before - I updated the comments, graphs and file there and am posting the links here: https://github.com/PediatricOpenTargets/ticket-tracker/issues/129
